src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -62,6 +63,11 @@ public ApplicationInsightsServiceOptions()
         /// </summary>
         public bool EnableDebugLogger { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the currently authenticated user is added to telemetries or not.
+        /// </summary>
+        public bool EnableAuthenticationTracking { get; set; }
```

Comment:
can you make it explicit that this is applicable to server side telemetry only.

Reply:
Just pushed an update with a more detailed description of the option.